app/models/user.rb

```diff
@@ -3,4 +3,7 @@ class User < ActiveRecord::Base
   # :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable, :confirmable
+  has_many :books
```

Comment:
That's what the donation and order tables are for:  The owner of a donation
is a donor.  The owner of an order is a recipient.  I think that's
unambiguous:  so you can have users who are both donors and recipients.
So a user has_many donations and a user has_many orders ... but a user
doesn't have books directly.

Does that fix this?

On Tue, Feb 10, 2015 at 10:14 PM, Aaron Panchal <notifications@github.com>
wrote:

> In app/models/user.rb
> <https://github.com/hammersmith/bookends/pull/7#discussion_r24476065>:
>
> > @@ -3,4 +3,7 @@ class User < ActiveRecord::Base
> >    # :lockable, :timeoutable and :omniauthable
> >    devise :database_authenticatable, :registerable,
> >           :recoverable, :rememberable, :trackable, :validatable, :confirmable
> > +  has_many :books
>
> This is confusing -- we may need to finesse how a user is linked to books.
> What if the same user donates and requests?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hammersmith/bookends/pull/7/files#r24476065>.
>



-- 

​--------
​-------===============########===============---------------​


Want to quickly check
​a ​
system status
​ or​
report
​a problem to the IT team?
Use http://justme.westmont.edu
​
​
Have a problem that requires tracking and IT email response?
Use http://mayday.westmont.edu​